server/integration_tests/test_data/e2e_triple/whattranscriptsdoesithave/chart_config.json

```diff
@@ -0,0 +1,98 @@
+{
+  "client": "test_detect-and-fulfill",
+  "config": {
+    "categories": [
+      {
+        "blocks": [
+          {
+            "columns": [
+              {
+                "tiles": [
+                  {
+                    "answerTableTileSpec": {
+                      "columns": [
+                        {
+                          "header": "->hasRNATranscript",
+                          "propertyExpr": "->hasRNATranscript"
+                        }
+                      ]
+                    },
+                    "entities": [
+                      "bio/FGFR1",
+                      "bio/galGal5_FGFR1",
+                      "bio/galGal6_FGFR1",
+                      "bio/hg19_FGFR1",
+                      "bio/hg38_FGFR1",
+                      "bio/mm10_Fgfr1",
+                      "bio/mm9_Fgfr1"
+                    ],
+                    "title": "The hasRNATranscript for FGFR1, FGFR1 and 5 more other are as follows:",
+                    "type": "ANSWER_TABLE"
+                  }
+                ]
+              }
+            ]
+          }
+        ]
+      }
+    ]
+  },
+  "context": {},
+  "debug": {},
+  "entities": [
+    {
+      "dcid": "bio/FGFR1",
+      "name": "FGFR1",
+      "type": ""
+    },
+    {
+      "dcid": "bio/galGal5_FGFR1",
+      "name": "FGFR1",
+      "type": ""
+    },
+    {
+      "dcid": "bio/galGal6_FGFR1",
+      "name": "FGFR1",
+      "type": ""
+    },
+    {
+      "dcid": "bio/hg19_FGFR1",
+      "name": "FGFR1",
+      "type": ""
+    },
+    {
+      "dcid": "bio/hg38_FGFR1",
+      "name": "FGFR1",
+      "type": ""
+    },
+    {
+      "dcid": "bio/mm10_Fgfr1",
+      "name": "Fgfr1",
+      "type": ""
+    },
+    {
+      "dcid": "bio/mm9_Fgfr1",
+      "name": "Fgfr1",
+      "type": ""
+    }
+  ],
+  "pastSourceContext": "",
+  "place": {},
+  "placeFallback": {},
+  "placeSource": "UNKNOWN",
+  "places": [],
+  "relatedThings": {
+    "childPlaces": {},
+    "childTopics": [],
+    "exploreMore": {},
+    "mainTopics": [],
+    "parentPlaces": [],
+    "parentTopics": [],
+    "peerPlaces": [],
+    "peerTopics": []
+  },
+  "svSource": "UNKNOWN",
+  "userMessages": [
+    "See relevant information for FGFR1 and Fgfr1 based on the previous query."
```

Comment:
hmm, same name (data issue?)

Reply:
ah yes, needed to filter out the duplicates, updated